quirks/change-password-URLs.json

```diff
@@ -1,13 +1,17 @@
 {
     "500px.com": "https://web.500px.com/settings/account/security",
     "acorns.com": "https://app.acorns.com/settings/change-password",
+    "allianz.com.br": "https://www.allianz.com.br/alteracao-de-password-ecliente",
     "amctheatres.com": "https://www.amctheatres.com/amcstubs/account",
+    "anatel.gov.br": "https://apps.anatel.gov.br/AnatelConsumidor/ConsumidorEditar.aspx",
     "atlassian.com": "https://id.atlassian.com/manage-profile/security",
     "autodesk.com": "https://accounts.autodesk.com/Profile/Security",
     "censys.io": "https://censys.io/account",
+    "claro.com.br": "https://minhanet.net.com.br/webcenter/portal/MinhaNet/pages_alterarsenha",
     "cloudflare.com": "https://dash.cloudflare.com/profile/authentication",
     "consumidor.gov.br": "https://www.consumidor.gov.br/pages/usuario/editar",
     "digitalocean.com": "https://cloud.digitalocean.com/settings/security",
+    "discord.com": "https://discord.com/channels/@me",
```

Comment:
This URL doesn’t seem to get me much further along the process of changing my password. It just loaded to the Discord home for me when logged in.

Reply:
good point @rmondello ✂️